### DISCUSSION TITLE
Auto backup feature

### DISCUSSION THREAD

**Discussion by Rafid41** [lnreader/lnreader#596] [Ideas]
Can automatic backup feature be added?
It will be a great feature to save progress. I forget to make manual backup many times.

**Comment by ZeroKun265** [comment]
No response after a while
Tachiyomi already has support for automatic backups and I already set up auto-upload on my nextcloud server
It would be amazing to have automatic backups on this as well since I already lost my progress once after my phone stopped working and I didn't remember where I left off

**Comment by rajarsheechatterjee** [comment]
Create an [Feature Request](https://github.com/LNReader/lnreader/issues/new?assignees=&labels=Feature+Request&projects=&template=request_feature.yml) if there isn't an existing issue for it.
